packages/experience-builder-sdk/src/hooks/useBreakpoints.ts

```diff
@@ -46,14 +50,19 @@ const getFallbackBreakpointIndex = (breakpoints: Breakpoint[]) => {
 export const getValueForBreakpoint = (
   valuesByBreakpoint: ValuesByBreakpoint,
   breakpoints: Breakpoint[],
-  activeBreakpointIndex: number
+  activeBreakpointIndex: number,
+  variableName: string
 ) => {
   const fallbackBreakpointIndex = getFallbackBreakpointIndex(breakpoints);
   const fallbackBreakpointId = breakpoints[fallbackBreakpointIndex].id;
   if (valuesByBreakpoint instanceof Object) {
     // Assume that the values are sorted by media query to apply the cascading CSS logic
     for (let index = activeBreakpointIndex; index >= 0; index--) {
       const breakpointId = breakpoints[index].id;
+      if (AvailableDesignTokenVariables.has(variableName)) {
+        if (variableName === 'cfMargin' || variableName === 'cfPadding')
```

Comment:
The if statements could be combined
```suggestion
      if (AvailableDesignTokenVariables.has(variableName) && ['cfMargin',  'cfPadding'].includes(variableName)) {
```

Reply:
I purposely did this so that when there's other attributes like border or whatever, I will add a separate clause for that so that I can direct that to the `getDesignTokenRegistrationForBorder` function. So this is the structure I want for the future even though today, in this PR you can shorten it like this. Hopefully what I wrote makes sense

Reply:
```
      if (AvailableDesignTokenVariables.has(variableName)) {
        if (variableName === 'cfMargin' || variableName === 'cfPadding')
          return getDesignTokenRegistrationForSpacing(valuesByBreakpoint[breakpointId]);
        if(variableName === 'cfBorder')return 
          getDesignTokenRegistrationForBorder(valuesByBreakpoint[breakpointId]);
      }
```

Reply:
Got it, makes sense 👍 

If enough values are checked on the inner part, a `switch` might be preferable too